### PR TITLE
Form example

### LINKED ISSUE #24: form data example

### PR DESCRIPTION
**Pull request by casperin**
Examples on how to handle form submits

Closes #24

### CODE REVIEW

**Comment by casperin** [comment]
Not specifically, but vim just does that for me. Like it did for https://github.com/actix/actix-web/pull/362

**Comment by DoumanAsh** [comment]
It seems somewhat similar to [JSON example](https://github.com/actix/examples/tree/master/json) so maybe we'd need some common example on extractors in future.
 But anyway it seems to be good as it is.

One thin I'd like you to do is to ad `README.md` file with short description how to use it and what it is.

File: form/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[package]
+name = "form-example"
+version = "0.1.0"
+authors = ["Gorm Casper <gcasper@gmail.com>"]
+
+[dependencies]
+actix = "0.5"
+actix-web = "^0.6"
+serde = "1.0.69"
```
Comment (DoumanAsh) [inline form/Cargo.toml]: Didn't notice it at first glance...

Please use `1.0` for serde versions

**Comment by casperin** [comment]
I definitely looked to the json example for how to structure it, although I didn't agree with using postman for testing. Maybe, with a merge, `curl` could be used as an example of requests.

**Comment by DoumanAsh** [comment]
Ah this is understandable. I actually prefer simplicity so I see nothing wrong with your approach.

Last thing: can you please add check of your example in `.travis.yml`?

**Comment by casperin** [comment]
Totally missed the travis file. It's done :)

**Comment by DoumanAsh** [comment]
Yep, Thank you

**Pull request merged**